fog/view/enclave/impl/src/lib.rs

```diff
@@ -46,6 +46,40 @@ struct DecryptedMultiViewStoreQueryResponse {
     block_range: BlockRange,
 }
 
+/// Helper struct that contains block data for the client `QueryResponse`
+#[derive(Clone)]
+struct BlockData {
+    /// The highest processed block count that will be returned to the client.
+    highest_processed_block_count: u64,
+    /// The timestamp for the highest processed block count
+    highest_processed_block_signature_timestamp: u64,
+    /// The last known block count that will be returned to the client.
+    last_known_block_count: u64,
```

Comment:
:question: I'm guessing this is coming from `UntrustedQueryResponse` which could clarify some
1. Is the timestamp meant to be generic or does it have units like milliseconds?
2. I'm guessing _last known_ means _highest_ that we know exists but we haven't been able to decode or whatever versus highest processed which is the _highest_ block that we've been able to muck with and can provide meaningful input on

Reply:
1. Yea it's coming from UntrustedQueryResponse. 
2. Yes.